src/resources/js/views/month-mobile-events.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Month Mobile Events Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.monthMobileEvents = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.events.views.manager
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		calendar: '.tribe-events-calendar-month',
+		calendarDay: '.tribe-events-calendar-month__day-cell--mobile',
+		calendarDaySelected: '.tribe-events-calendar-month__day-cell--selected',
+		mobileEventsDay: '.tribe-events-calendar-month-mobile-events__mobile-day',
+	};
+
+	/**
+	 * Closes all mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of view container
+	 *
+	 * @return {void}
+	 */
+	obj.closeAllEvents = function( $container ) {
+		$container.find( obj.selectors.calendarDay ).each( function( index, header ) {
+			var $header = $( header );
+			var contentId = $header.attr( 'aria-controls' );
+
+			/**
+			 * Define empty jQuery object in the case contentId is false or undefined
+			 * so that we don't get selectors like #false or #undefined.
+			 * Also only perform accordion actions if header has aria-controls attribute.
+			 */
+			var $content = $( '' );
+			if ( contentId ) {
+				$content = $container.find( '#' + contentId );
+				tribe.events.views.accordion.closeAccordion( $header, $content );
+			}
+
+			obj.closeMobileEvents( $header, $content );
+		} );
+	};
+
+	/**
+	 * Opens mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $header jQuery object of mobile day button
+	 * @param {jQuery} $content jQuery object of mobile events container
+	 *
+	 * @return {void}
+	 */
+	obj.openMobileEvents = function( $header, $content ) {
+		// only perform accordion actions if $header has aria-controls attribute.
+		var contentId = $header.attr( 'aria-controls' );
+		if ( contentId ) {
+			tribe.events.views.accordion.openAccordion( $header, $content );
+		}
+
+		$header.addClass( obj.selectors.calendarDaySelected.className() );
+		$content.parent().css( 'display', 'block' );
```

Comment:
Should we control this with a class? Makes it more flexible.
Also, should we target the parent always?

Reply:
When you ask if we should always target the parent, do you mean in `accordion.js`? I'm targeting the parent here because it's specific to mobile events js, the accordion doesn't always have this structure.

Can do to toggle a class here.